CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- Added ability to get metrics from VertX (@yuri-zubov sponsored by Actility, https://www.actility.com)
+- Added health-check VertX (@yuri-zubov sponsored by Actility, https://www.actility.com)
+
+### Security
+- updated yard dependency to `~> 0.9.11` per: https://nvd.nist.gov/vuln/detail/CVE-2017-17042 (@yuri-zubov sponsored by Actility, https://www.actility.com)
+- updated rubocop dependency to `~> 0.51.0` per: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-8418. (@yuri-zubov sponsored by Actility, https://www.actility.com)
+
+### Breaking Changes
```

Comment:
Generally I like to see the order of changes from most important to least important:
- breaking changes
- security
- fixed
- added
- removed

I will fix this up post acceptance prior to release if there are no other changes requested.